pir/proto/payload.proto

```diff
@@ -28,8 +28,11 @@ message Request {
   // Each query may have 1 or more ciphertexts.
   Ciphertexts query = 1;
 
-  // Galois keys
-  bytes keys = 2;
+  // Galois keys, needed to compute substitution operation on ciphertexts.
+  bytes galois_keys = 2;
+
+  // Relinearization keys, only needed for recursion depths more than 1.
```

Comment:
If I'm not mistaken, note that you may not _necessarily_ need the relinearization keys: you could leave the ciphertext grow in size. 
The current implementation choice seems to requires their use?